base/io.jl

```diff
@@ -740,15 +752,31 @@ mutable struct EachLine
 end
 
 """
-    eachline(stream::IO=STDIN; chomp::Bool=true)
+    eachline(io::IO=STDIN; chomp::Bool=true)
     eachline(filename::AbstractString; chomp::Bool=true)
 
 Create an iterable `EachLine` object that will yield each line from an I/O stream
-or a file. Iteration calls `readline` on the stream argument repeatedly with
+or a file. Iteration calls [`readline`](@ref) on the stream argument repeatedly with
 `chomp` passed through, determining whether trailing end-of-line characters are
 removed. When called with a file name, the file is opened once at the beginning of
 iteration and closed at the end. If iteration is interrupted, the file will be
 closed when the `EachLine` object is garbage collected.
+
+# Examples
+```jldoctest
+julia> open("my_file.txt", "w") do io
+           write(io, "JuliaLang is a GitHub organization.\\n It has many members.\\n");
+       end;
+
+julia> el = eachline("my_file.txt");
+
+julia> for li in el
```

Comment:
`li` -> `line` since that is what you use on the next row. Also, perhaps skip the `el` variable and write this as
```
julia> for line in eachline("my_file.txt")
           print(line)
       end
```

Reply:
fixed, thanks for the review <3